Tests/swiftplotTests/SwiftPlotTestCase.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+@testable import SwiftPlot
+
+class SwiftPlotTestCase: XCTestCase {
+  // TODO: Possibly allow setting this via a command-line flag?
```

Comment:
It would be nice if it were possible. I wasn't able to find a way to do this, but the same thing happens when we set the path using --package-path.